public/javascripts/SVLabel/src/SVLabel/task/Task.js

```diff
@@ -255,6 +255,7 @@ function Task (geojson, tutorialTask, currentLat, currentLng, startPointReversed
     this.complete = function () {
         status.isComplete = true;
         properties.completedByAnyUser = true;
+        let oldPriority = properties.priority;
```

Comment:
I think this might be left over from an old implementation or debugging? Not seeing it used anywhere else.